### PR TITLE
Change default ad position for Windows on release channel

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -594,6 +594,52 @@
                 "platform": ["WINDOWS"]
             }
         },
+        {
+            "name": "BraveAds.ShowCustomAdNotificationOnWindowsStudy",
```
Comment (goodov) [inline seed/seed.json:598]: looks like we already have a study with the same name for `release` channel. shouldn't we change it instead?
https://github.com/brave/brave-variations/blob/8f6da1c1078b6f04da1b34c1501b1b68d20ec0c4/seed/seed.json#L679-L705

Comment (tmancey) [reply seed/seed.json:598]: @goodov the three studies have different filters:
```
            "filter": {
                "min_version": "91.1.26.37",
                "max_version": "92.1.30.18",
                "channel": ["NIGHTLY", "BETA"],
                "platform": ["WINDOWS"]
            }


            "filter": {
                "min_version": "92.1.30.19",
                "min_os_version": "10.0.17134.*",
                "channel": ["BETA", "NIGHTLY"],
                "platform": ["WINDOWS"]
            }


            "filter": {
                "min_version": "92.1.30.19",
                "channel": ["RELEASE"],
                "platform": ["WINDOWS"]
            }
```

**Comment by kjozwiak** [comment]
We'll wait till the above feedback/concerns from @goodov are addressed. Once we're looking good, we can merge this into `main` and QA will run through it before merging into `production`.

File: seed/seed.json (hunk 1 above)
Comment (tmancey) [reply seed/seed.json]: Fixed

**Comment by tmancey** [comment]
Closing issue as reverting rollout of custom ad notifications due to community feedback

**Pull request closed without merging**